build/commands/scripts/commands.js

```diff
@@ -260,5 +260,9 @@ program
   .option('--full', 'format all lines in changed files instead of only the changed lines')
   .action(util.format)
 
+program
+  .command('mass-rename')
```

Comment:
I don't think we should change the names of the utilities here, that just makes things confusing imo. Let's stick with move-source-file

Reply:
The original name is mass-rename.py, so all is correct .

Reply:
I see, it calls mass-rename which calls move_source_file. I got confused because the changes were on move_source_file